training/distributed_training/pytorch/data_parallel/rnnt/RNNT_notebook.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>Typos: <code>prarameters</code> ﻿, <code>traning</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>LibriSpeech dataset seems not approved by Legal for public commercial use. I'll sync up with you offline.</p>

Reply:
<p>Submitted a ticket for a legal approval </p>

Reply:
ticket is approved! I added a link to the license in the notebook